hera_cal/apply_cal.py

```diff
@@ -192,6 +196,10 @@ def apply_cal(data_infilename, data_outfilename, new_calibration, old_calibratio
         flag_file: optional path to file containing flags to be ORed with flags in input data. Must have
             the same shape as the data.
         flag_filetype: filetype of flag_file to pass into io.load_flags. Either 'h5' (default) or legacy 'npz'.
+        a_priori_ex_ants_yaml : path to YAML with antenna flagging information parsable by
```

Comment:
Why are we ignoring frequency/time flags? Wouldn't that be useful?

Also, be very careful with ex_ants when `redundant_solution=True`, since you might flag baselines that are actually good but have excluded antennas in the "exemplar" baseline for the redundant group.

Reply:
I can add time / frequency flags.

Reply:
I think that the way I'm flagging ex_ants, (setting flags of ex_ants True but not excluding them from reds) should make sure there aren't issues with removing the exemplary baseline as a key to the red groups.